ddtrace/llmobs/_llmobs.py

```diff
@@ -560,7 +561,7 @@ def _tag_embedding_io(cls, span, input_documents=None, output_text=None):
                 span.set_tag_str(OUTPUT_VALUE, output_text)
             else:
                 try:
-                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text))
+                    span.set_tag_str(OUTPUT_VALUE, json.dumps(output_text, default=_unserializable_default_repr))
                 except TypeError:
                     log.warning("Failed to parse output text. Output text must be JSON serializable.")
```

Comment:
How about moving this warning log inside the `_unserializable_default_repr` function?

Something like:
```
def _unserializable_default_repr(obj):
    log.warning("Failed to parse I/O. I/O must be JSON serializable. Defaulting to X".)
    return "[Unserializable object: {}]".format(repr(obj))
```

Reply:
I like the idea of adding a warning log to inside the unserializable_default_repr() method, but this warning log still needs to be there in cases the object is actually unserializable (and its default repr() method is also not serializable())